third_party/tonic/logging/dart_error.cc

```diff
@@ -10,28 +10,15 @@
 namespace tonic {
 namespace DartError {
 const char kInvalidArgument[] = "Invalid argument.";
-DartPersistentValue on_error;
-UnhandledExceptionReporter log_unhandled_exception = [](const std::string&,
-                                                        const std::string&) {};
+UnhandledExceptionReporter log_unhandled_exception = [](Dart_Handle,
```

Comment:
The c++ style guide disallows global variables that have non-trivial destructors: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Could you use a function pointer instead?

Reply:
Done